include/zephyr/storage/flash_map.h

```diff
@@ -118,7 +118,7 @@ int flash_area_check_int_sha256(const struct flash_area *fa,
  * @p ID is unknown, it will be NULL on output.
  *
  * @return  0 on success, -EACCES if the flash_map is not available ,
```

Comment:
Fix typo in commit message: `storeage` -> `storage`

Reply:
ROTFL